### ISSUE TITLE
How to know ECC is enabled?

### ISSUE THREAD

**Issue by VarunBatraIT** [kopia/kopia#4282]
Hi
Is there anyway to know if ECC is enabled (error correction code) in repository?

I tried following:
```
kopia repository status
cat /home/codevarun/.config/kopia/repository.config
kopia policy show --global
```
I am pretty sure that I created with ecc enabled and it also displyaed it is enabled immediately when I created a repository.

But now I wonder, if it is that same drive or a different one. 

```
kopia repository status
Config file:         /home/codevarun/.config/kopia/repository.config

Description:         Repository in Filesystem: /media/codevarun/Asus-Backup/Kopia/
Hostname:            workstation
Username:            codevarun
Read-only:           false
Format blob cache:   15m0s

Storage type:        filesystem
Storage capacity:    5 TB
Storage available:   3.2 TB
Storage config:      {
                       "path": "/media/codevarun/Asus-Backup/Kopia/",
                       "fileMode": 384,
                       "dirMode": 448,
                       "dirShards": null
                     }

Unique ID:           baa4f74beb40b04c277b3d7024620a4cd7b402274faf06ef6924fa54f12e087a
Hash:                BLAKE2B-256-128
Encryption:          AES256-GCM-HMAC-SHA256
Splitter:            DYNAMIC-4M-BUZHASH
Format version:      3
Content compression: true
Password changes:    true
Max pack length:     21 MB
Index Format:        v2

Epoch Manager:       enabled
Current Epoch: 0

Epoch refresh frequency: 20m0s
Epoch advance on:        20 blobs or 10.5 MB, minimum 24h0m0s
Epoch cleanup margin:    4h0m0s
Epoch checkpoint every:  7 epochs
```
```
cat /home/codevarun/.config/kopia/repository.config


{
  "storage": {
    "type": "filesystem",
    "config": {
      "path": "/media/codevarun/Asus-Backup/Kopia/",
      "fileMode": 384,
      "dirMode": 448,
      "dirShards": null
    }
  },
  "caching": {
    "cacheDirectory": "../../.cache/kopia/da56c316ce034ef5",
    "maxCacheSize": 5242880000,
    "maxMetadataCacheSize": 5242880000,
    "maxListCacheDuration": 30
  },
  "hostname": "workstation",
  "username": "codevarun",
  "description": "Repository in Filesystem: /media/codevarun/Asus-Backup/Kopia/",
  "enableActions": false,
  "formatBlobCacheDuration": 900000000000
}%                                                                                                                                                                                            

```
```
kopia policy show --global

Policy for (global):

Retention:
  Annual snapshots:                        0   (defined for this target)
  Monthly snapshots:                       0   (defined for this target)
  Weekly snapshots:                        0   (defined for this target)
  Daily snapshots:                         0   (defined for this target)
  Hourly snapshots:                        0   (defined for this target)
  Latest snapshots:                       50   (defined for this target)
  Ignore identical snapshots:           true   (defined for this target)

Files policy:
  Ignore cache directories:             true   (defined for this target)
  No ignore rules:
  Read ignore rules from files:                (defined for this target)
    .kopiaignore
  Scan one filesystem only:            false   (defined for this target)

Error handling policy:
  Ignore file read errors:             false   (defined for this target)
  Ignore directory read errors:        false   (defined for this target)
  Ignore unknown types:                 true   (defined for this target)

Scheduling policy:
  Scheduled snapshots:
    None.
  Manual snapshot:                     false   (defined for this target)

Uploads:
  Max parallel snapshots (server/UI):      1   (defined for this target)
  Max parallel file reads:                 -   (defined for this target)
  Parallel upload above size:         2.1 GB   (defined for this target)

Compression:
  Compressor:                     s2-default   (defined for this target)
  Compress files regardless of extensions.
  Compress files of all sizes.

Metadata compression:
  Compressor:                   zstd-fastest   (defined for this target)

Splitter:
  Algorithm override:   (repository default)   (defined for this target)

No actions defined.

OS-level snapshot support:
  Volume Shadow Copy:                  never   (defined for this target)

Logging details (0-none, 10-maximum):
  Directory snapshotted:                   5   (defined for this target)
  Directory ignored:                       5   (defined for this target)
  Entry snapshotted:                       0   (defined for this target)
  Entry ignored:                           5   (defined for this target)
  Entry cache hit:                         0   (defined for this target)
  Entry cache miss:                        0   (defined for this target)
```

**Comment by kronenpj** [comment]
Running `kopia repository status --json` shows ECC information under the `contentFormat` key. For example, when run on my repository I get:
```json
{
    "configFile": "/home/kopia/.config/kopia/repository.config",
    "uniqueIDHex": "edf0e1b15fbef7e46d185f2964bb4786f95229f82a70a179fa3325952c14b43c",
    "clientOptions": {
        "hostname": "backup",
        "username": "kopia",
        "description": "Repository in Filesystem: /export/home1/kopia",
        "enableActions": false,
        "formatBlobCacheDuration": 900000000000
    },
    "storage": {
        "type": "filesystem",
        "config": {
            "path": "/export/home1/kopia",
            "fileMode": 416,
            "dirMode": 488,
            "dirShards": null
        }
    },
    "volume": {
        "capacity": 2000290414592,
        "available": 759628181504
    },
    "contentFormat": {
        "hash": "BLAKE3-256",
        "encryption": "AES256-GCM-HMAC-SHA256",
        "ecc": "REED-SOLOMON-CRC32",
        "eccOverheadPercent": 2,
        "version": 3,
        "maxPackSize": 20971520,
        "indexVersion": 2,
        "epochParameters": {
            "Enabled": true,
            "EpochRefreshFrequency": 1200000000000,
            "FullCheckpointFrequency": 7,
            "CleanupSafetyMargin": 14400000000000,
            "MinEpochDuration": 86400000000000,
            "EpochAdvanceOnCountThreshold": 20,
            "EpochAdvanceOnTotalSizeBytesThreshold": 10485760,
            "DeleteParallelism": 4
        },
        "enablePasswordChange": true
    },
    "objectFormat": {
        "splitter": "DYNAMIC-1M-BUZHASH"
    },
    "blobRetention": {}
}
```

**Comment by VarunBatraIT** [comment]
Good enough for me thanks a lot.
